examples/budgetting-method/README.md

```diff
@@ -0,0 +1,62 @@
+# Budgeting method
+
+OpenSLO specifies three types of budgeting methods:
```

Comment:
Could the folder be renamed to have a single "t" in "budgeting" to align with the text?

Reply:
Yes! Sorry, for that forget to look on file names :D